sycl/test/e2e_test_requirements/no-xfail-without-tracker.cpp

```diff
@@ -27,7 +27,7 @@
 // - ...and check if the list of improperly XFAIL-ed tests needs to be updated.
 //
 // RUN: grep -rI "XFAIL:" %S/../../test-e2e \
-// RUN: -A 1 --include=*.c --include=*.cpp --no-group-separator | \
```

Comment:
I suggest that we also update `sycl/test-e2e/lit.cfg.py` to exclude `.c` suffix from being considered as a test